src/pages/questions/ProductInformation.jsx

```diff
@@ -75,6 +77,11 @@ const ProductInformation = ({ question }) => {
     };
   }, [question?.barcode]);
 
+  const handleHideImages = (event) => {
+    setHideImages(event.target.checked);
+    localSettings.update(localSettingsKeys.hideImages, event.target.checked);
+  }
```

Comment:
Not for this PR, but it let me think that a hook `useSettingState` which is similar to `React.useState`+save data in settings could be interesting

Reply:
yeah, that would be useful